client/src/app/ui/modules/sorting/modules/sorting-tree/components/sorting-tree/sorting-tree.component.ts

```diff
@@ -559,16 +568,73 @@ export class SortingTreeComponent<T extends Identifiable & Displayable> implemen
         };
     }
 
+    /**
+     * Handles a click on a row. If the control key is clicked, the element is
+     * added/removed from a multiselect list /(which will be handled on
+     * dropping)
+     *
+     * @param event MouseEvent.
+     * @param indx The index of the row clicked.
+     */
+    public onItemClick(event: MouseEvent | Event, indx: number): void {
+        if (event.type === `click` || (<KeyboardEvent>event).key === ` `) {
+            event.preventDefault();
+            if ((<MouseEvent>event).ctrlKey) {
```

Comment:
While you are at it - this might be a nice addition to prevent text selection when using multiselect. 
```suggestion
            if ((<MouseEvent>event).ctrlKey) {
                document.getSelection().removeAllRanges();
```